lib/chef/provider/mount/aix.rb

```diff
@@ -149,6 +180,18 @@ def enable_fs
           end
         end
 
+        def mount_options_unchanged?
+          current_resource_options = @current_resource.options
+          current_resource_options.pop
```

Comment:
It feels a little magical. a comment and something like `current_resource_options.delete_if { |x| x == "rw" }` might be more clear.